### DISCUSSION TITLE
[Marketplace] Label for items having a No/Low Code configurator

### DISCUSSION THREAD

**Discussion by peppemanzi** [mia-platform/community#347] [Ideas]
### Which area of this project could be most improved?

Marketplace

### Description

I think that Marketplace items that come with a graphical configurator should be highlighted with a tag/label.

For example if I'm looking for an API Gateway in the Marketplace, it would be nice to know that Envoy and NGINX based API Gateways are configurable in No Code mode thanks to a Console extension.

**Comment by akelity** [comment]
This feature has been positively evaluated and may be considered for inclusion in the roadmap in the future, depending on priorities and overall specific needs.
